tests/test_converter.py

```diff
@@ -1416,3 +1416,30 @@ def test_diff_meas_circuit(self):
 
         qtemp2 = load(qc, measurements=[qml.expval(qml.PauliZ(0))])
         assert qtemp()[0] != qtemp2()[0] and qtemp2()[0] == qml.expval(qml.PauliZ(0))
+
+    def test_direct_qnode_ui(self):
+        """Test the UI where the loaded function is passed directly to qml.QNode
+        along with a device"""
+
+        dev = qml.device("default.qubit")
+        angle = Parameter("angle")
+
+        qc = QuantumCircuit(2, 2)
+        qc.rx(angle, [0])
+        qc.measure(0, 0)
+        qc.rx(angle, [1])
+        qc.cz(0, 1)
+
+        measurements = [qml.expval(qml.PauliZ(0)), qml.vn_entropy([1])]
+
+        @qml.qnode(dev)
+        def circuit_native_pennylane(angle):
+            qml.RX(angle, wires=0)
+            qml.measure(0)
+            qml.RX(angle, wires=1)
+            qml.CZ([0, 1])
+            return qml.expval(qml.PauliZ(0)), qml.vn_entropy([1])
+
+        qnode = qml.QNode(load(qc, measurements), dev)
```

Comment:
I would prefer to use `qml.from_qiskit` here, since that's the actual intended UI, but we run the tests with the released version of PL, so `qml.from_qiskit` can't take the measurements argument.